pkg/chunk/schema.go

```diff
@@ -529,6 +539,39 @@ func (entries v7Entries) GetWriteEntries(bucket Bucket, metricName model.LabelVa
 }
 
 func (v7Entries) GetReadQueries(bucket Bucket) ([]IndexQuery, error) {
+	// Replaced with v8Schema series index
+	return nil, ErrNoMetricNameNotSupported
+}
+
+// v8Entries is the same as v7Entries however with a series index instead of a metric name index
```

Comment:
`v8Entries` embedding `v6Entries` is correct. The comment says it's the same as `v7` because we are replacing the index, with other indexes saying the same, however this comment is confusing because v8Entries GetWriteEntries is completely different.

I will update the comment - thanks